File: BLH.ApproveIQ.Portal/ClientApp/src/app/data/constants/invoice-constants.ts
export class InvoiceStatuses {
  static readonly New = 'New';
  static readonly PendingApproval = 'PendingApproval';
  static readonly Approved = 'Approved';
  static readonly Rejected = 'Rejected';
  static readonly Paid = 'Paid';
  static readonly OnHold = 'On Hold';
}

export class ApprovalActions {
  static readonly Assigned = 'Assigned';
  static readonly Approved = 'Approved';
  static readonly Rejected = 'Rejected';
  static readonly Reassigned = 'Reassigned';
  static readonly PutOnHold = 'Put On Hold';
  static readonly Released = 'Released';
}

File: BLH.ApproveIQ.Domain/Constants/InvoiceConstants.cs
namespace BLH.ApproveIQ.Domain.Constants;

public static class InvoiceStatuses
{
    public const string New = "New";
    public const string PendingApproval = "PendingApproval";
    public const string Approved = "Approved";
    public const string Rejected = "Rejected";
    public const string Paid = "Paid";
    public const string OnHold = "On Hold";
}

public static class ApprovalActions
{
    public const string Assigned = "Assigned";
    public const string Approved = "Approved";
    public const string Rejected = "Rejected";
    public const string Reassigned = "Reassigned";
    public const string PutOnHold = "Put On Hold";
    public const string Released = "Released";
}
